apps/ideaspace/src/components/modules/SaveIdea/SaveIdea.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { atoms, molecules } from '@devlaunchers/components/src/components';
+import SaveButton, { SaveButtonProps } from '@devlaunchers/components/src/components/molecules/InteractionButtons/SaveButton';
+import { Item } from 'semantic-ui-react';
+
+const SaveIdea = ({savedCards, setSavedCards, id}) =>{
+    
+    return(
+        <SaveButton
+            style = {{
```

Comment:
can we move this to the `StyledSaveIdea.js` file which is currently empty? 

Something like:

`import styled from "styled-components";

export const StyledSaveIdea = styled.SaveIdea`
// custom styles go here
`;
`

Reply:
Sure, I will move the code to the StyledSaveIdea.js file. 
Thanks for the suggestion!

Reply:
The updates have been implemented.